### PR TITLE
Add basic tests for Book

### PR DESCRIPTION
**Pull request by shlomo-maghen**

### CODE REVIEW

File: src/main/java/bookup/Book.java [hunk 1]
```diff
@@ -14,11 +14,11 @@ public Book(String isbn) {
 		this.isbn = isbn;
 	}
 
-	public String getisbn() {
+	public String getISBN() {
```
Comment (aharonmaghen) [inline src/main/java/bookup/Book.java:17]: So NOW we're doing capital?!

File: src/main/java/bookup/Book.java [hunk 2]
```diff
@@ -64,8 +64,8 @@ public void setImageURL(URL url) {
 
 	@Override
 	public String toString() {
-		return "ISBN: " + isbn + "\nTITLE: " + title + "\nAUTHOR: " + author + 
-		"\nPAGES: " + pages + "\nYEAR: " + year;
+		return String.format("ISBN: %s, Title: %s, Author: %s, Pages: %s, Year: %s, Image URL: %s",
```
Comment (aharonmaghen) [inline src/main/java/bookup/Book.java]: I like it!

File: src/main/java/bookup/Book.java [hunk 3]
```diff
@@ -64,8 +64,8 @@ public void setImageURL(URL url) {
...
+				isbn, title, author, pages, year, imageURL);
 	}
```
Comment (aharonmaghen) [inline src/main/java/bookup/Book.java:69]: I like it!

File: src/main/java/bookup/Book.java (hunk 1 above)
Comment (shlomo-maghen) [reply src/main/java/bookup/Book.java:17]: If anything should at least be `getIsbn()`. I would be ok with that.

File: src/main/java/bookup/Book.java (hunk 3 above)
Comment (shlomo-maghen) [reply src/main/java/bookup/Book.java:69]: 😄

**Comment by shlomo-maghen** [comment]
.

**Pull request merged**